src/mumble/CMakeLists.txt

```diff
@@ -454,18 +466,21 @@ else()
 
 		target_sources(mumble
 			PRIVATE
-				"GlobalShortcut_unix.cpp"
-				"GlobalShortcut_unix.h"
 				"Log_unix.cpp"
 				"os_unix.cpp"
 		)
 
 		if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 			find_library(LIB_RT rt)
 			target_link_libraries(mumble PRIVATE ${LIB_RT})
-		endif()
 
-		target_link_libraries(mumble PRIVATE X11::Xext)
+			# TODO: check on evdev/udev or linux/input.h presence instead of Linux as it's also present in FreeBSD
+			target_sources(mumble
+				PRIVATE
+					"GlobalShortcut_linux.cpp"
+					"GlobalShortcut_linux.h"
+			)
```

Comment:
Will there be any conflicts between the shortcut engines in case X11 is enabled as well?

Reply:
Currently there is namespace conflicts but I think there should be the possibility of having both X11 and evdev shortcuts in the same build.

Reply:
So this is being worked on? :thinking: